src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -351,3 +368,11 @@ def __update_ui(self, robot_status: RobotStatus) -> None:
         )
 
         self.battery_progress_bar.setValue(power_status.battery_voltage)
+
+    def __calculate_average_round_trip_time(self, new_time: float) -> int:
+        self.previous_primitive_rtt_values.append(new_time)
+        average_rtt_time_milliseconds = 0
+        for value in self.previous_primitive_rtt_values:
+            average_rtt_time_milliseconds += value
```

Comment:
nit: you can probably simplify this to
```suggestion
        sum_rtt_time_milliseconds = sum(self.previous_primitive_rtt_values)
```